### ISSUE TITLE
how do i use.

### ISSUE THREAD

**Issue by ghost** [monkeyman192/BoxBoyLevelEditor#17]
i have no idea on how to use this so can someone help me?

**Comment by SomeKitten** [comment]
Yes, I can help you. How far have you gotten in trying to use this? Would you prefer a compiled .exe version? (I'm not sure how easy it is to make this in particular into an .exe but I can try)

**Comment by ghost** [comment]
Yes Please! I would greatly appreciate that!

**Comment by TheMasterCoders** [comment]
Well, poop my pants. It never happened.
